doc/ref_model/chapters/glossary.md

```diff
@@ -80,14 +78,24 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Container Image**:	Stored instance of a container that holds a set of software needed to run an application.
 - **Container**:	A lightweight and portable executable image that contains software and all of its dependencies.
+>_*Note:*_ OCI defines **Container** as "An environment for executing processes with configurable isolation and resource limitations. For example, namespaces, resource limits, and mounts are all part of the container environment." 
+A **Container** provides operating-system-level virtualization by abstracting the “user space”. One big difference between **Containers** and **VM**s is that unlike VMs, where each **VM** is self-contained with all the operating systems components are within the **VM** package, containers "share" the host system’s kernel with other containers.
+
+- **Container Runtime**: reads the configuration files for a **Container** from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions."
+>_*Note:*_ adapted from [OCI Glossary](https://github.com/opencontainers/runtime-spec/blob/master/glossary.md).
+
+- **Container Engine**: Software components used to create, destroy, and manage containers on top of an operating system.
+
 - **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
-- **Kubernetes Cluster**: A set of machines, called nodes, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master node.
-- **Kubernetes Master**:	The master node(s) manages the worker nodes and the pods in the cluster. Multiple master nodes are used to provide a cluster with failover and high availability.
+- **Kubernetes Cluster**: A set of worker machines, called nodes, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
+>_*Note:*_ adapted from [Kubernetes Glossary](https://kubernetes.io/docs/reference/glossary/?all=true#term-cluster).
+- **Kubernetes Master**:	The master(s) manage the worker nodes and the pods in the cluster. Multiple masters are used to provide a cluster with failover and high availability.
```

Comment:
Can we change back to original definition `master(s)` > `master node(s)` to remain inline with Kubernetes definition please: https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster

> The master node(s) manages the worker nodes and the pods in the cluster. Multiple master nodes are used to provide a cluster with failover and high availability.

Reply:
@tomkivlin Unfortunately, the glossary defines "node" as "A node is a worker machine in Kubernetes." Hence, using the term "master node" is confusing at best.

Reply:
Thanks. I also like that better.